Woof/Woof/View/DetailedPetSitterView/RaitingView.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+/// A view displaying a rating of the person.
+struct RatingView: View {
```

Comment:
Do we need all this stuff to just display a simple view?
For me, it looks like the view that handles the rating process, does not just display the info.
Do you understand every line in this file and what does it do?

Reply:
Yes, agree.Simplified the view.